Fails with unrelated pylint error:
```
adafruit_bluefruitspi.py:250:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
```

--- a/adafruit_bluefruitspi.py
+++ b/adafruit_bluefruitspi.py
@@ -247,7 +247,7 @@ class BluefruitSPI:
                 return rsp
             raise RuntimeError("Unknown response (id:{0})".format(hex(msgid)))
         except RuntimeError as error:
-            raise RuntimeError("AT command failure: " + repr(error))
+            raise RuntimeError("AT command failure: " + repr(error)) from error
 
     def command_check_OK(self, command, delay=0.0):  # pylint: disable=invalid-name
         """Send a fully formed bytestring AT command, and check

--- a/examples/bluefruitspi_neocolorpicker.py
+++ b/examples/bluefruitspi_neocolorpicker.py
@@ -5,8 +5,8 @@ import time
 import busio
 import board
 from digitalio import DigitalInOut
-from adafruit_bluefruitspi import BluefruitSPI
 import neopixel
+from adafruit_bluefruitspi import BluefruitSPI
 
 ADVERT_NAME = b"BlinkaNeoLamp"
 

